src/devices/Ccs811/samples/Program.cs

```diff
@@ -50,7 +51,7 @@
 else if (platformChoice.KeyChar == '2')
 {
     WriteLine("Creating an instance of a CCS811 using FT4222 drivers.");
-    using (var i2cBus = FtCommon.GetDevices()[0].CreateI2cBus())
+    using (var i2cBus = new Ft4222Device(FtCommon.GetDevices()[0]).CreateI2cBus())
```

Comment:
could FtCommon enumerate both Ft4222 and Ft232? This way we could create a device regardless which device someone is using